backend/server.js

```diff
@@ -42,14 +41,16 @@ const hapi = new Hapi.Server({
 sbp('okTurtles.data/set', SERVER_INSTANCE, hapi)
 
 sbp('sbp/selectors/register', {
+  'backend/server/broadcastEntry': async function (entry: GIMessage) {
+    const pubsub = sbp('okTurtles.data/get', PUBSUB_INSTANCE)
+    const pubsubMessage = createMessage(NOTIFICATION_TYPE.ENTRY, entry.serialize())
+    const subscribers = pubsub.enumerateSubscribers(entry.contractID())
+    console.log(chalk.blue.bold(`[pubsub] Broadcasting ${entry.description()}`))
```

Comment:
Didn't you mention that there was a way for the `GIMessage` object to get logged with `entry.description()` automatically when being logged?

If so, wouldn't it be possible to remove this `console.log` line and then remove `'backend/server/broadcastEntry'`, resulting in the same number of log lines as before, but automated? Or was there a reason you did it this way instead?

Reply:
Well I noticed than the default console output format was also being used in many places, likely providing valuable debug information in the build output, for example:
```js
 GIMessage {
    _decrypted: undefined,
    _mapping: {
      key: '21XWnNHXUB2pLXf3Y1wZ2aJny2sZGz4Hd4zrACUuA77tZqn9tC',
      value: '{"message":"{\\"version\\":1,\\"previousHEAD\\":\\"21XWnNUqk6LEdCAQ1jQTY5Tro1YPMx5DxAmXewwK295JT9qqo5\\",\\"contractID\\":\\"21XWnNUqk6LEdCAQ1jQTY5Tro1YPMx5DxAmXewwK295JT9qqo5\\",\\"op\\":[\\"ae\\",\\"{\\\\\\"type\\\\\\":\\\\\\"gi.contracts/identity/process\\\\\\",\\\\\\"data\\\\\\":{\\\\\\"attributes\\\\\\":{\\\\\\"username\\\\\\":\\\\\\"Bob-568\\\\\\",\\\\\\"email\\\\\\":\\\\\\"bob@okturtles.com\\\\\\"}},\\\\\\"meta\\\\\\":{}}\\"],\\"nonce\\":0.34964236633868184}","sig":{"type":"default","sig":"21XWnNN5w9SZ7kGvZBH3e2XwUSXXWEVGfkgbSSLWcZNAvdrj7i"}}'
    },
    _message: {
      version: 1,
      previousHEAD: '21XWnNUqk6LEdCAQ1jQTY5Tro1YPMx5DxAmXewwK295JT9qqo5',
      contractID: '21XWnNUqk6LEdCAQ1jQTY5Tro1YPMx5DxAmXewwK295JT9qqo5',
      op: [Array],
      nonce: 0.34964236633868184
    }
  }
  ```

Now if `entry.description()` is made to be the default logging format like I mentioned it was possible to do, then it will be automatically used everywhere by default, and that debug information will be lost.

Unless we implement and use another logging function for callsites where more debug information shoul be displayed.

Reply:
It would be interesting to identify a clean minimal subset of debug info that should be included so that GIMessage console output can be improved.

Reply:
Hmm. OK, you make a fair point. Resolving this, let's leave it as-is.